stdlib/number.gr

```diff
@@ -577,12 +705,16 @@ provide let sin = (radians: Number) => {
   }
 }
 
+// TODO(#1478): Improve the examples with more accurate values
 /**
  * Computes the cosine of a number (in radians) using Chebyshev polynomials.
  *
  * @param radians: The input in radians
  * @returns The computed cosine
  *
+ * @example Number.cos(Number.pi) == -0.9999998749960952
```

Comment:
I don't know if we should be showcasing that our `cos` implementation has flaws :laughing:

Reply:
I agree, I think the best option is to merge: #2032 as a temporary solution to our inaccuracies, and then fix this example.